cafy_pytest/plugin.py

```diff
@@ -338,6 +344,7 @@ def pytest_configure(config):
     CafyLog.mongomode=config.option.mongo_mode
     CafyLog.giso_dir = config.option.giso_dir
     script_list = config.option.file_or_dir
+    CafyLog.collection = config.option.collection
```

Comment:
If this CafyLog.collection variable is not going to be used in cafykit, then this can be defined as a local variable or try to directly access config.option.collection